### ISSUE TITLE
2021-08

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#11]

**Comment by qiaojy19** [comment]
# Aug - 2021

## 08.23 - 08.27
| Title                                                                                                                                     |
| ----------------------------------------------------------------------------------------------------------------------------------------- |
| [Aegis: A Trusted, Automatic and Accurate Verification Framework for Vertical Federated Learning      ](https://arxiv.org/abs/2108.06958) |
| [Back to the Drawing Board: A Critical Evaluation of Poisoning Attacks on Federated Learning          ](https://arxiv.org/abs/2108.10241) |
| [Byzantine Fault-Tolerance in Federated Local SGD under 2f-Redundancy                                 ](https://arxiv.org/abs/2108.11769) |
| [Canoe : A System for Collaborative Learning for Neural Nets                                          ](https://arxiv.org/abs/2108.12124) |
| [Data-Free Evaluation of User Contributions in Federated Learning                                     ](https://arxiv.org/abs/2108.10623) |
| [Dropout against Deep Leakage from Gradients                                                          ](https://arxiv.org/abs/2108.11106) |
| [Enabling SQL-based Training Data Debugging for Federated Learning                                    ](https://arxiv.org/abs/2108.11884) |
| [Federated Learning for Open Banking                                                                  ](https://arxiv.org/abs/2108.10749) |
| [Federated Learning for Privacy-Preserving Open Innovation Future on Digital Health                   ](https://arxiv.org/abs/2108.10761) |
| [Federated Learning for UAV Swarms Under Class Imbalance and Power Consumption Constraints            ](https://arxiv.org/abs/2108.10748) |
| [Federated Learning Meets Fairness and Differential Privacy                                           ](https://arxiv.org/abs/2108.09932) |
| [Federated Multi-Task Learning under a Mixture of Distributions                                       ](https://arxiv.org/abs/2108.10252) |
| [Federated Reinforcement Learning: Techniques, Applications, and Open Challenges                      ](https://arxiv.org/abs/2108.11887) |
| [PIVODL: Privacy-preserving vertical federated learning over distributed labels                       ](https://arxiv.org/abs/2108.11444) |
| [Private Multi-Task Learning: Formulation and Applications to Federated Learning                      ](https://arxiv.org/abs/2108.12978) |
| [Targeting Underrepresented Populations in Precision Medicine: A Federated Transfer Learning Approach ](https://arxiv.org/abs/2108.12112) |
## 08.16 - 08.20
| Title                                                                                                                                         |
| --------------------------------------------------------------------------------------------------------------------------------------------- |
| [A Novel Attribute Reconstruction Attack in Federated Learning                                            ](https://arxiv.org/abs/2108.06910) |
| [Accelerating Federated Learning with a Global Biased Optimiser                                           ](https://arxiv.org/abs/2108.09134) |
| [Aegis: A Trusted, Automatic and Accurate Verification Framework for Vertical Federated Learning          ](https://arxiv.org/abs/2108.06958) |
| [Aggregation Delayed Federated Learning                                                                   ](https://arxiv.org/abs/2108.07433) |
| [Anarchic Federated Learning                                                                              ](https://arxiv.org/abs/2108.09875) |
| [Blockchain-based Trustworthy Federated Learning Architecture                                             ](https://arxiv.org/abs/2108.06912) |
| [Client Selection Approach in Support of Clustered Federated Learning over Wireless Edge Networks         ](https://arxiv.org/abs/2108.08768) |
| [Communication-Efficient Federated Learning via Robust Distributed Mean Estimation                        ](https://arxiv.org/abs/2108.08842) |
| [Cross-Silo Federated Learning for Multi-Tier Networks with Vertical and Horizontal Data Partitioning     ](https://arxiv.org/abs/2108.08930) |
| [Detection of Illicit Drug Trafficking Events on Instagram: A Deep Multimodal Multilabel Learning Approach](https://arxiv.org/abs/2108.08920) |
| [Efficient Federated Meta-Learning over Multi-Access Wireless Networks                                    ](https://arxiv.org/abs/2108.06453) |
| [Fair and Consistent Federated Learning                                                                   ](https://arxiv.org/abs/2108.08435) |
| [Federated Distributionally Robust Optimization for Phase Configuration of RISs                           ](https://arxiv.org/abs/2108.09026) |
| [Federated Learning with Correlated Data: Taming the Tail for Age-Optimal Industrial IoT                  ](https://arxiv.org/abs/2108.07504) |
| [Federated Multi-Target Domain Adaptation                                                                 ](https://arxiv.org/abs/2108.07792) |
| [Federated Variational Learning for Anomaly Detection in Multivariate Time Series                         ](https://arxiv.org/abs/2108.08404) |
| [FedSkel: Efficient Federated Learning on Heterogeneous Systems with Skeleton Gradients Update            ](https://arxiv.org/abs/2108.09081) |
| [Fed-TGAN: Federated Learning Framework for Synthesizing Tabular Data                                     ](https://arxiv.org/abs/2108.07927) |
| [Fine-tuning is Fine in Federated Learning                                                                ](https://arxiv.org/abs/2108.07313) |
| [Flexible Clustered Federated Learning for Client-Level Data Distribution Shift                           ](https://arxiv.org/abs/2108.09749) |
| [Learning Federated Representations and Recommendations with Limited Negatives                            ](https://arxiv.org/abs/2108.07931) |
| [Mobility-Aware Cluster Federated Learning in Hierarchical Wireless Networks                              ](https://arxiv.org/abs/2108.09103) |
| [Multi-Center Federated Learning                                                                          ](https://arxiv.org/abs/2108.08647) |
| [Multi-task Federated Learning for Heterogeneous Pancreas Segmentation                                    ](https://arxiv.org/abs/2108.08537) |
| [Order Optimal One-Shot Federated Learning for non-Convex Loss Functions                                  ](https://arxiv.org/abs/2108.08677) |
| [Personalised Federated Learning: A Combinational Approach                                                ](https://arxiv.org/abs/2108.09618) |
| [Regulating Ownership Verification for Deep Neural Networks: Scenarios, Protocols, and Prospects          ](https://arxiv.org/abs/2108.09065) |
| [SemiFed: Semi-supervised Federated Learning with Consistency and Pseudo-Labeling                         ](https://arxiv.org/abs/2108.09412) |
| [Towards More Efficient Federated Learning with Better Optimization Objects                               ](https://arxiv.org/abs/2108.08577) |
| [Towards Secure and Practical Machine Learning via Secret Sharing and Random Permutation                  ](https://arxiv.org/abs/2108.07463) |
| [Wireless Federated Langevin Monte Carlo: Repurposing Channel Noise for Bayesian Sampling and Privacy     ](https://arxiv.org/abs/2108.07644) |
## 08.09 - 08.13
| Title                                                                                                                                        |
| -------------------------------------------------------------------------------------------------------------------------------------------- |
| [A Contract Theory based Incentive Mechanism for Federated Learning                                      ](https://arxiv.org/abs/2108.05568) |
| [A Credibility-aware Swarm-Federated Deep Learning Framework in Internet of Vehicles                     ](https://arxiv.org/abs/2108.03981) |
| [ABC-FL: Anomalous and Benign client Classification in Federated Learning                                ](https://arxiv.org/abs/2108.04551) |
| [An Operator Splitting View of Federated Learning                                                        ](https://arxiv.org/abs/2108.05974) |
| [Collaborative Unsupervised Visual Representation Learning from Decentralized Data                       ](https://arxiv.org/abs/2108.06492) |
| [Communication Optimization in Large Scale Federated Learning using Autoencoder Compressed Weight Updates](https://arxiv.org/abs/2108.05670) |
| [Dynamic Attention-based Communication-Efficient Federated Learning                                      ](https://arxiv.org/abs/2108.05765) |
| [FedMatch: Federated Learning Over Heterogeneous Question Answering Data                                 ](https://arxiv.org/abs/2108.05069) |
| [FedPAGE: A Fast Local Stochastic Gradient Method for Communication-Efficient Federated Learning         ](https://arxiv.org/abs/2108.04755) |
| [FedPara: Low-rank Hadamard Product Parameterization for Efficient Federated Learning                    ](https://arxiv.org/abs/2108.06098) |
| [Joint Optimization in Edge-Cloud Continuum for Federated Unsupervised Person Re-identification          ](https://arxiv.org/abs/2108.06493) |
| [Personalized Federated Learning with Clustering: Non-IID Heart Rate Variability Data Application        ](https://arxiv.org/abs/2108.01903) |
| [Reducing the Communication Cost of Federated Learning through Multistage Optimization                   ](https://arxiv.org/abs/2108.06869) |
| [SAFE: Secure Aggregation with Failover and Encryption                                                   ](https://arxiv.org/abs/2108.05475) |
## 08.02 - 08.06

| Title                                                                                                                                                                |
| -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Building a Foundation for Data-Driven, Interpretable, and Robust Policy Design using the AI Economist                           ](https://arxiv.org/abs/2108.02904) |
| [Communication-Efficient Federated Learning via Predictive Coding                                                                ](https://arxiv.org/abs/2108.00918) |
| [Decentralized Federated Learning with Unreliable Communications                                                                 ](https://arxiv.org/abs/2108.02397) |
| [Decentralized Federated Learning: Balancing Communication and Computing Costs                                                   ](https://arxiv.org/abs/2107.12048) |
| [Determining Sentencing Recommendations and Patentability Using a Machine Learning Trained Expert System                         ](https://arxiv.org/abs/2108.04088) |
| [Evaluating Federated Learning for Intrusion Detection in Internet of Things: Review and Challenges                              ](https://arxiv.org/abs/2108.00974) |
| [Fed-BEV: A Federated Learning Framework for Modelling Energy Consumption of Battery Electric Vehicles                           ](https://arxiv.org/abs/2108.04036) |
| [FederatedNILM: A Distributed and Privacy-preserving Framework for Non-intrusive Load Monitoring based on Federated Deep Learning](https://arxiv.org/abs/2108.03591) |
| [FedJAX: Federated learning simulation with JAX                                                                                  ](https://arxiv.org/abs/2108.02117) |
| [FedLab: A Flexible Federated Learning Framework                                                                                 ](https://arxiv.org/abs/2107.11621) |
| [GIFAIR-FL: An Approach for Group and Individual Fairness in Federated Learning                                                  ](https://arxiv.org/abs/2108.02741) |
| [Information Stealing in Federated Learning Systems Based on Generative Adversarial Networks                                     ](https://arxiv.org/abs/2108.00701) |
| [Multi-task Federated Edge Learning (MtFEEL) in Wireless Networks                                                                ](https://arxiv.org/abs/2108.02517) |
| [On Addressing Heterogeneity in Federated Learning for Autonomous Vehicles Connected to a Drone Orchestrator                     ](https://arxiv.org/abs/2108.02712) |
| [Personalized Federated Learning with Clustering: Non-IID Heart Rate Variability Data Application                                ](https://arxiv.org/abs/2108.01903) |
| [PPT: A Privacy-Preserving Global Model Training Protocol for Federated Learning in P2P Networks                                 ](https://arxiv.org/abs/2105.14408) |
| [Secure and Privacy-Preserving Federated Learning via Co-Utility                                                                 ](https://arxiv.org/abs/2108.01913) |
| [Secure Neuroimaging Analysis using Federated Learning with Homomorphic Encryption                                               ](https://arxiv.org/abs/2108.03437) |
| [The Effect of Training Parameters and Mechanisms on Decentralized Federated Learning based on MNIST Dataset                     ](https://arxiv.org/abs/2108.03508) |
| [User Scheduling for Federated Learning Through Over-the-Air Computation                                                         ](https://arxiv.org/abs/2108.02891) |
